spec/controllers/users_controller_spec.rb

```diff
@@ -21,12 +21,32 @@
       it "redirects to the sign in page" do
         expect(response).to redirect_to sign_in_path
       end
-    end
 
-    context "email sending" do
+      it "makes the user follow the inviter" do
+        luke = Fabricate(:user)
+        invitation = Fabricate(:invitation, inviter: luke, recipient_email: 'hans@falcon.com')
+        post :create, user: { email: 'hans@falcon.com', full_name: 'Hans Solo', password: 'password'}, invitation_token: invitation.token
+        hans = User.where(email: 'hans@falcon.com').first
+        expect(hans.follows?(luke)).to be_truthy
+      end
 
-      before { ActionMailer::Base.deliveries.clear }
+      it "make the inviter follow the user" do
+        luke = Fabricate(:user)
+        invitation = Fabricate(:invitation, inviter: luke, recipient_email: 'hans@falcon.com')
+        post :create, user: { email: 'hans@falcon.com', full_name: 'Hans Solo', password: 'password'}, invitation_token: invitation.token
```

Comment:
Line is too long. [137/80]<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.